### PR TITLE
Backport v0.31 add read range blocks admin cmd

### PR DESCRIPTION
**Pull request by zhangchiqing**
Backport https://github.com/onflow/flow-go/pull/4451

### CODE REVIEW

**Review by pattyshack — APPROVED** [review]
assuming you backport all the changes

File: admin/commands/storage/read_transactions.go [hunk 1]
```diff
@@ -69,36 +70,27 @@ func findUint64(input map[string]interface{}, field string) (uint64, error) {
 	return uint64(val), nil
 }
 
-// Validator validates the request.
-// Returns admin.InvalidAdminReqError for invalid/malformed requests.
-func (c *GetTransactionsCommand) Validator(req *admin.CommandRequest) error {
-	input, ok := req.Data.(map[string]interface{})
+func findString(input map[string]interface{}, field string) (string, error) {
```
Comment (peterargue) [inline admin/commands/storage/read_transactions.go:73]: maybe move this to `helpers.go`?

**Review by peterargue — APPROVED** [review]
looks good for v0.31, but please address these comments in a followup PR on master

File: admin/commands/storage/read_range_blocks.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+	lights, err := read.ReadBlockLightByHeightRange(c.blocks, data.startHeight, data.endHeight)
+	if err != nil {
+		return nil, err
+	}
+	return commands.ConvertToInterfaceList(lights)
+}
+
+func (c *ReadRangeBlocksCommand) Validator(req *admin.CommandRequest) error {
+	return nil
```
[the remark this answers is not in the document]
Comment (zhangchiqing) [reply admin/commands/storage/read_range_blocks.go:38]: https://github.com/onflow/flow-go/issues/4467

**Pull request merged**